packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -1661,11 +1700,39 @@ export class Vector3 {
         return s;
     }
 
+	/**
+	 * Gets the rectangular coordinates of a Vector3 from its polar coordinates
+	 * @param theta defines the angle for the vertical
+	 * @param phi defines the angle for the horizontal
+	 * @param length defines the length of the Vector3 (in polar coordinate space)
+	 * @returns the rectangular coordinate Vector3
+	 */
+	public static FromPolar(theta: Number, phi: Number, length: Number): Vector3{
+		let x = length * Math.sin(theta) * Math.cos(phi);
+		let y = length * Math.sin(theta) * Math.sin(phi);
+		let z = length * Math.cos(theta);
+		return new Vector3(x, y, z);
+	}
+
+	/**
+	* Get rotation between two vectors
+	* @param origin defines the starting point
+	* @param target defines the ending point
+	* @returns the rotation between the vectors
+	*/
+	public static GetRotationBetweenVectors(origin: DeepImmutable<Vector3>, target: DeepImmutable<Vector3>): Vector3{

```

Comment:
what is the difference with getAngleBetweenVectors ? it sounds confusing to have both ?

Reply:
getAngleBetweenVectors returns only 1 angle, relative to a plane/normal. GetRotationBetweenVectors on the other hand returns both angles, without a normal.

For example, setting the rotation of a Node to the values returned by giving GetRotationBetweenVectors the node's position and a target point would have it facing the target point.

So, getAngleBetweenVectors will only return the angle relative to *one* given axis, while GetRotationBetweenVectors returns the angle**s** relative to all 3 axes.